crates/bevy_ui/src/layout/ui_surface.rs

```diff
@@ -30,18 +31,41 @@ impl From<taffy::NodeId> for LayoutNode {
     }
 }
 
+pub(crate) struct BevyTaffyTree<T>(TaffyTree<T>);
```

Comment:
The name's a bit too long and ugly, maybe just:
```suggestion
pub(crate) struct UiTree<T>(TaffyTree<T>);
```
would be better, or something like that.